.gitignore

```diff
@@ -40,3 +40,4 @@ uploads
 
 # Mac files
 .DS_Store
+foodaving/.DS_Store
```

Comment:
Maybe `**/.DS_Store` would be a better expression as it will match in all directories?
https://github.com/git/git/blob/master/Documentation/RelNotes/1.8.2.txt#L85

Also, you need to `git rm` the DS_Store files that you already added.